Aug-03 14:28:29.007 [main] DEBUG nextflow.cli.Launcher - $> nextflow run main.nf
Aug-03 14:28:29.147 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 24.04.4
Aug-03 14:28:29.193 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/home/gitpod/.nextflow/plugins; core-plugins: nf-amazon@2.5.3,nf-azure@1.6.1,nf-cloudcache@0.4.1,nf-codecommit@0.2.1,nf-console@1.1.3,nf-ga4gh@1.3.0,nf-google@1.13.2-patch1,nf-tower@1.9.1,nf-wave@1.4.2-patch1
Aug-03 14:28:29.204 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Aug-03 14:28:29.205 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Aug-03 14:28:29.209 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.12.0 in 'deployment' mode
Aug-03 14:28:29.223 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Aug-03 14:28:29.322 [main] DEBUG nextflow.cli.CmdRun - Applied DSL=2 from script declaration
Aug-03 14:28:29.340 [main] DEBUG nextflow.cli.CmdRun - Launching `main.nf` [confident_dijkstra] DSL2 - revision: 6f78aeee15
Aug-03 14:28:29.371 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
Aug-03 14:28:29.371 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins resolved requirement=[]
Aug-03 14:28:29.443 [main] DEBUG nextflow.Session - Session UUID: b755cc54-055d-4c50-85e7-399913f5f048
Aug-03 14:28:29.444 [main] DEBUG nextflow.Session - Run name: confident_dijkstra
Aug-03 14:28:29.444 [main] DEBUG nextflow.Session - Executor pool size: 16
Aug-03 14:28:29.478 [main] DEBUG nextflow.file.FilePorter - File porter settings maxRetries=3; maxTransfers=50; pollTimeout=null
Aug-03 14:28:29.486 [main] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'FileTransfer' minSize=10; maxSize=48; workQueue=LinkedBlockingQueue[10000]; allowCoreThreadTimeout=false
Aug-03 14:28:29.518 [main] DEBUG nextflow.cli.CmdRun - 
  Version: 24.04.4 build 5917
  Created: 01-08-2024 07:05 UTC 
  System: Linux 6.1.91-060191-generic
  Runtime: Groovy 4.0.21 on OpenJDK 64-Bit Server VM 11.0.23+9-LTS
  Encoding: UTF-8 (UTF-8)
  Process: 9944@daleor0917-protpipeline-op6d5or758g [127.0.0.1]
  CPUs: 16 - Mem: 62.8 GB (26.5 GB) - Swap: 375 GB (374.9 GB)
Aug-03 14:28:29.575 [main] DEBUG nextflow.Session - Work-dir: /workspace/Prot_Pipeline/Prot_Pipeline/work [xfs]
Aug-03 14:28:29.576 [main] DEBUG nextflow.Session - Script base path does not exist or is not a directory: /workspace/Prot_Pipeline/Prot_Pipeline/bin
Aug-03 14:28:29.597 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
Aug-03 14:28:29.614 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Aug-03 14:28:29.677 [main] DEBUG nextflow.cache.CacheFactory - Using Nextflow cache factory: nextflow.cache.DefaultCacheFactory
Aug-03 14:28:29.689 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 17; maxThreads: 1000
Aug-03 14:28:29.792 [main] DEBUG nextflow.Session - Session start
Aug-03 14:28:30.643 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Aug-03 14:28:30.679 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /home/gitpod/.nextflow/secrets/store.json
Aug-03 14:28:30.688 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@3b0c3951] - activable => nextflow.secret.LocalSecretsProvider@3b0c3951
Aug-03 14:28:30.774 [main] DEBUG nextflow.executor.ExecutorFactory - << taskConfig executor: null
Aug-03 14:28:30.775 [main] DEBUG nextflow.executor.ExecutorFactory - >> processorType: 'local'
Aug-03 14:28:30.781 [main] DEBUG nextflow.executor.Executor - [warm up] executor > local
Aug-03 14:28:30.788 [main] DEBUG n.processor.LocalPollingMonitor - Creating local task monitor for executor 'local' > cpus=16; memory=62.8 GB; capacity=16; pollInterval=100ms; dumpInterval=5m
Aug-03 14:28:30.791 [main] DEBUG n.processor.TaskPollingMonitor - >>> barrier register (monitor: local)
Aug-03 14:28:31.008 [main] DEBUG nextflow.Session - Workflow process names [dsl2]: buildDockerImage
Aug-03 14:28:31.008 [main] DEBUG nextflow.Session - Igniting dataflow network (1)
Aug-03 14:28:31.009 [main] DEBUG nextflow.processor.TaskProcessor - Starting process > buildDockerImage
Aug-03 14:28:31.010 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
  Script_d2b72e62ef232332: /workspace/Prot_Pipeline/Prot_Pipeline/main.nf
Aug-03 14:28:31.011 [main] DEBUG nextflow.script.ScriptRunner - > Awaiting termination 
Aug-03 14:28:31.011 [main] DEBUG nextflow.Session - Session await
Aug-03 14:28:31.393 [Task submitter] DEBUG n.executor.local.LocalTaskHandler - Launch cmd line: /bin/bash -ue .command.run
Aug-03 14:28:31.396 [Task submitter] INFO  nextflow.Session - [0f/6b404e] Submitted process > buildDockerImage
Aug-03 14:28:31.717 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 1; name: buildDockerImage; status: COMPLETED; exit: 1; error: -; workDir: /workspace/Prot_Pipeline/Prot_Pipeline/work/0f/6b404ed684c52b0483f95bddf3311e]
Aug-03 14:28:31.718 [Task monitor] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'TaskFinalizer' minSize=10; maxSize=48; workQueue=LinkedBlockingQueue[10000]; allowCoreThreadTimeout=false
Aug-03 14:28:31.725 [TaskFinalizer-1] DEBUG nextflow.processor.TaskProcessor - Handling unexpected condition for
  task: name=buildDockerImage; work-dir=/workspace/Prot_Pipeline/Prot_Pipeline/work/0f/6b404ed684c52b0483f95bddf3311e
  error [nextflow.exception.ProcessFailedException]: Process `buildDockerImage` terminated with an error exit status (1)
Aug-03 14:28:31.749 [TaskFinalizer-1] ERROR nextflow.processor.TaskProcessor - Error executing process > 'buildDockerImage'

Caused by:
  Process `buildDockerImage` terminated with an error exit status (1)


Command executed:

  docker build -t blastp_image -f Dockerfile_blastp .

Command exit status:
  1

Command output:
  (empty)

Command error:
  #0 building with "default" instance using docker driver
  
  #1 [internal] load build definition from Dockerfile_blastp
  #1 transferring dockerfile: 2B done
  #1 DONE 0.0s
  
  #2 [internal] load .dockerignore
  #2 transferring context: 2B done
  #2 DONE 0.0s
  ERROR: failed to solve: failed to read dockerfile: open /workspace/.docker-root/tmp/buildkit-mount2402291902/Dockerfile_blastp: no such file or directory

Work dir:
  /workspace/Prot_Pipeline/Prot_Pipeline/work/0f/6b404ed684c52b0483f95bddf3311e

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`
Aug-03 14:28:31.756 [main] DEBUG nextflow.Session - Session await > all processes finished
Aug-03 14:28:31.770 [TaskFinalizer-1] DEBUG nextflow.Session - Session aborted -- Cause: Process `buildDockerImage` terminated with an error exit status (1)
Aug-03 14:28:31.788 [Task monitor] DEBUG n.processor.TaskPollingMonitor - <<< barrier arrives (monitor: local) - terminating tasks monitor poll loop
Aug-03 14:28:31.788 [main] DEBUG nextflow.Session - Session await > all barriers passed
Aug-03 14:28:31.797 [main] DEBUG n.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0; failedCount=1; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=312ms; cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=1; peakCpus=1; peakMemory=0; ]
Aug-03 14:28:31.975 [main] DEBUG nextflow.cache.CacheDB - Closing CacheDB done
Aug-03 14:28:31.991 [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye
